addons/mail/tests/test_mail_channel.py

```diff
@@ -45,113 +45,199 @@ def setUpClass(cls):
         cls.test_group = Channel.create({
             'channel_type': 'group',
             'name': 'Group',
+            'public': 'private'})
+        # Chat
+        cls.test_chat = Channel.create({
```

Comment:
name suggestion: `chat_user1(_user2)` (where user(s) are specified, we shouldn't create an empty chat)

possibly use `channel_get` to create it